src/components/malta/AddStudentSection.css

```diff
@@ -0,0 +1,61 @@
+.profile-container2 {
```

Comment:
Add in subfolder of malta 
example:
src/components/malta/AddStudent/AddStudentSection.css